### DISCUSSION TITLE
No encuentro la opción modificar opciones en TAREAS

### DISCUSSION THREAD

**Discussion by luisreymer** [chamilo/chamilo-lms#5473] [General]
Tengo la versión Chamilo LMS 1.11.26, al momento de crear tareas, me he dado cuenta de que los profesores pueden cambiar la opción de caducidad, estoy buscando la opción para limitar esta opción y no la encuentro!

**Comment by LudiscapeXApi** [comment]
¿Podrías especificar a qué te refieres con "tareas"?
¿Qué herramienta y contexto estás mencionando?
